wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/expression/values/EngineFunctionEvaluators.scala

```diff
@@ -66,7 +66,12 @@ object EngineFunctionEvaluators {
         val tryResult = for {
           //validate
           read <- readFile(fileToRead, ioFunctionSet, fileSizeLimitationConfig.readLinesLimit)
-          lines = read.split(System.lineSeparator)
+          // The if statement below tests for file emptiness to handle different expected outputs based on file contents.
```

Comment:
Nit: comment answers "what?" but not "why?"

```suggestion
          // Users expect an empty file to return zero lines
```